draft-irtf-cfrg-vdaf.md

```diff
@@ -3848,12 +3920,13 @@ by a wrapper gadget, `QueryGadget`, listed in {{gadget-wrappers}}. This gadget
 also evaluates the gadget polynomial for each gadget invocation in order to
 produce the gadget's output. Then compute the wire polynomials from the
 recorded values.
+Note that these polynomials are in the value (Lagrange) representation
+by construction.
 
 Next, choose a random point `t` (parsed from the query randomness), evaluate
 each wire polynomial at `t`, and evaluate the gadget polynomial at `t`.
-Note that the wire polynomials are in the value (Lagrange) representation
-by construction. Faster polynomial evaluation in this representation is
-possible by using the algorithm by {{Faz25}} without resorting to
+Polynomial evaluation in the value (Lagrange) representation is
+faster by using the algorithm by {{Faz25}} without resorting to
 polynomial interpolation.
 The results are recorded in the verifier message passed to the decision
 algorithm, where the test is finished.
```

Comment:
In the next paragraph, we should probably change "The random point `t` MUST NOT be one of the fixed evaluation points used to interpolate the wire polynomials." to "... used to define the wire polynomials."